anaconda.spec.in

```diff
@@ -482,6 +482,31 @@ rm -rf \
 %{_prefix}/lib/dracut/modules.d/80%{name}/*
 %{_prefix}/libexec/anaconda/dd_*
 
+# Make sure to not overwrite user configurtion files
+%config /etc/anaconda/anaconda.conf
+%config /etc/anaconda/conf.d/00-do-nothing.conf
+%config /etc/anaconda/conf.d/01-still-do-nothing.conf
```

Comment:
It there a way to wildcard these?

Reply:
From what I was investigating so far - no :/

Reply:
Also I would like to avoid creating a script which will be doing this automatically as from my experience - maintenance of this script will consume more resources than updating spec file manually once a while :]